src/averageAges.js

```diff
@@ -15,11 +15,12 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArr = people.filter(x => x.sex === 'm'
```

Comment:
use more informative names
```suggestion
  const menArr = people.filter(person => person.sex === 'm'
```

Reply:
fix it everywhere